bigchaindb/commands/utils.py

```diff
@@ -3,18 +3,39 @@
 """
 
 import argparse
+import builtins
+import functools
 import multiprocessing as mp
 import subprocess
+import sys
 
 import rethinkdb as r
 from pymongo import uri_parser
 
 import bigchaindb
+import bigchaindb.config_utils
 from bigchaindb import backend
 from bigchaindb.common.exceptions import StartupError
 from bigchaindb.version import __version__
 
 
+def configure_bigchaindb(command):
```

Comment:
Also, it may be important to point out that the idea of the decorator was also to include logging configuration and setup. So the goal of the decorator goes beyond the functionalities of `autoconfigure`. @sohkai already did the work: https://github.com/bigchaindb/bigchaindb/pull/1219/commits/a6584daf78e6011c1410e78d463b578acf62c081#diff-8f049242225a1f66c973d24911b5179aR31